### PR TITLE
Update 2018 blog content files to move author details in front-matter

### PR DESCRIPTION
**Pull request by dipesh-rawat**
Split out from PR https://github.com/kubernetes/website/pull/45957

This PR extends the changes made in PR https://github.com/kubernetes/website/pull/45865 and implements those changes to all the 2018 blog files as per the recommendation in https://github.com/kubernetes/website/pull/45865#issuecomment-2054036103. The primary change involves relocating author details into the metadata within the front matter.

Useful Link
[Preview Blog Summary Page](https://deploy-preview-46361--kubernetes-io-main-staging.netlify.app/blog)

### CODE REVIEW

File: content/en/blog/_posts/2018-07-09-IPVS-In-Cluster-Load-Balancing.md [hunk 1]
```diff
@@ -2,10 +2,12 @@
 layout: blog
 title:  'IPVS-Based In-Cluster Load Balancing Deep Dive'
 date:   2018-07-09
+author: >
+  Jun Du(Huawei),
+  Haibin Xie(Huawei),
+  Wei Liang(Huawei)
```
Comment (sftim) [inline content/en/blog/_posts/2018-07-09-IPVS-In-Cluster-Load-Balancing.md:8]: (nit)
```suggestion
  Jun Du (Huawei),
  Haibin Xie (Huawei),
  Wei Liang (Huawei)
```

**Review by sftim — COMMENTED** [review]
I'd expect to see `author:` in the front matter for https://kubernetes.io/blog/2018/03/first-beta-version-of-kubernetes-1-10/ and for other articles published that year where there's an identifiable author / set of authors.

**Comment by dipesh-rawat** [comment]
The latest commit push in the PR reflects the changes based on the review comment https://github.com/kubernetes/website/pull/46361#pullrequestreview-2053964688

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**